Levels/SampleBase/SampleBase.prefab

```diff
@@ -1925,13 +1925,6 @@
                     "$type": "EditorInspectorComponent",
                     "Id": 154105298091518109
                 },
-                "Component_[15452140819580244001]": {
-                    "$type": "GenericComponentWrapper",
-                    "Id": 15452140819580244001,
-                    "m_template": {
-                        "$type": "FlyCameraInputComponent"
```

Comment:
The FlyCameraInputComponent is what allows the user to navigate the world in the server window. Is there a way we can keep this component but only activate it on the server?

Reply:
That's interesting. What if we did an entirely separate camera entity for the server? @AMZN-Gene might have some thoughts here.

Reply:
Done: https://github.com/o3de/o3de/issues/10633